src/components/ChecklistPanelContent.js

```diff
@@ -57,6 +66,14 @@ const ChecklistPanelContent = ( {
 					{ requiredLabel }
 				</p>
 			) }
+			{ ( ! shouldBlockPublish && ! isComplete ) && (
+				<ToggleControl
+					label={ __( 'Override checks', 'altis-publication-checklist' ) }
```

Comment:
I would maybe label this "Bypass checks" rather than "Override checks". Bypass is a little less technical-sounding.

Reply:
Or just "Skip checks" perhaps?

Reply:
```suggestion
					label={ __( 'Skip checks', 'altis-publication-checklist' ) }
```